capstone/src/main/webapp/script.js

```diff
@@ -176,46 +187,110 @@ function callback(results, status) {
   _showBlackOwnedBusiness = false;
 }
 
+/** Creates an animated marker for each result location
+   @param {Object} place location to set marker for
+ */
 function setMarker(place) {
   const marker = new google.maps.Marker({
-    map: map,
+    map: _map,
     position: place.geometry.location,
     animation: google.maps.Animation.DROP,
   });
-  addToDisplayPanel(place.name);
+  addToDisplayPanel(place);
 }
- 
-function addToDisplayPanel(name) {
-  const locationElement = document.getElementById("restaurant-results");
-  locationElement.appendChild(createLocationElement(name));
+
+/** Itemizes each result into the collapsible panel
+@param {Object} place location to add to results panel
+ */
+function addToDisplayPanel(place) {
+  // put call to get details here and get new place before adding to panel
+  const locationElement = document.getElementById('restaurant-results');
+  locationElement.appendChild(createLocationElement(place));
+  locationElement.appendChild(createAdditionalInfo(place));
 }
- 
-function createLocationElement(placeName) {
-  const locElement = document.createElement('p');
-  locElement.className = 'place';
- 
-  const mainElement = document.createElement('span');
-  mainElement.innerText = placeName;
- 
-  locElement.appendChild(mainElement);
-  return locElement;
+
+/** Toggles on and off additional info upon click */
+function displayAdditionalInfo() {
+  const coll = document.getElementsByClassName('collapsible');
+
+  for (let i = 0; i < coll.length; i++) {
+    coll[i].addEventListener('click', function() {
+      this.classList.toggle('active');
+      const content = this.nextElementSibling;
+      if (content.style.maxHeight) {
```

Comment:
Here, we're toggling open or closed on click, right?

Reply:
Yes, that's what's happening here.